src/fmeval/transforms/common.py

```diff
@@ -191,3 +195,75 @@ def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
+    """This Transform computes the maximum BertScore value given various possible targets from a record and
+    augments said record.
+    """
+
+    def __init__(
+        self,
+        target_output_keys: List[str],
+        model_output_keys: List[str],
+        output_keys: List[str],
+        allow_duplicate_input_keys: bool,
+        bertscore_model: Union[BertscoreHelperModel, ActorHandle],
+        target_output_delimiter: Optional[str] = "<OR>",
+    ):
+        """BertScoreMax initializer.
+        :param target_output_keys: The keys corresponding to target outputs.
+        :param model_output_keys: The keys corresponding to model outputs.
+        :param output_keys: The output keys for this Transform, which correspond
+            to the BERT scores that get computed.
+        :param allow_duplicate_input_keys: See docstring for SummarizationAccuracyMetric.
+        :param bertscore_model: A BertscoreHelperModel instance or a Ray actor handle for a BertscoreHelperModel.
+        :param target_output_delimiter: The delimiter used to separate the possible
+            target outputs within the `target_output` string.
+        """
+        super().__init__(
+            target_output_keys,
+            model_output_keys,
+            output_keys,
+            allow_duplicate_input_keys,
+            bertscore_model,
+            target_output_delimiter,
+        )
+        self.register_input_output_keys(
+            target_output_keys + model_output_keys,
+            output_keys,
+            allow_duplicates=allow_duplicate_input_keys,
+        )
+        self.target_output_keys = target_output_keys
+        self.model_output_keys = model_output_keys
+        self.bertscore_model = bertscore_model
+        self.target_output_delimiter = target_output_delimiter
+
+        # BertScore transform used to compute metrics
+        self.bert_score_transform = BertScore(
+            target_output_keys=self.target_output_keys,
+            model_output_keys=self.model_output_keys,
+            output_keys=[BERT_SCORE],
+            allow_duplicate_input_keys=allow_duplicate_input_keys,
+            bertscore_model=self.bertscore_model,
+        )
+
+    @validate_call
+    def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
+        """Augment the input record with a maximum BERT_SCORE value computed via BertScore.compute_metric.
+
+        :param record: The input record.
+        :returns: The input record with BERT_SCORE metric added in.
+        """
+
+        for target_output_key, model_output_key, output_key in zip(
+            self.target_output_keys, self.model_output_keys, self.output_keys
+        ):
+            # separating possible targets by target output delimiter to use for BertScore
+            possible_targets = record[target_output_key].split(self.target_output_delimiter)
+            scores = [
+                BertScore.compute_metric(self.bert_score_transform, target, record[model_output_key])
```

Comment:
This should be `self.bert_score_transform.compute_metric(target, record[model_output_key)`. You're calling an instance method, so you should use `instance.method_name(arg1, arg2, ...)` and not `ClassName.method_name(instance_reference, arg1, ...)`.